### PR TITLE
[RuboCop] Clean up the Rakefile

### PR DESCRIPTION
**Pull request by segiddins**

### CODE REVIEW

File: Rakefile [hunk 1]
```diff
@@ -640,39 +625,37 @@ def git_branch_list(arguments = nil)
...
 end
 
 def changelog_for_repo(repo, version)
   changelog_path = File.expand_path('CHANGELOG.md', repo)
-  if File.exists?(changelog_path)
+  if File.exist?(changelog_path)
     title_token = '## '
     current_verison_title = title_token + version.to_s
-    text = File.open(changelog_path, "r:UTF-8") { |f| f.read }
+    text = File.open(changelog_path, 'r:UTF-8', &:read)
```
Comment (kylef) [inline Rakefile]: This descriptor is never closed.

Comment (segiddins) [inline Rakefile]: It is, since it's using the block version

Comment (kylef) [inline Rakefile]: Sorry, my bad.

**Comment by kylef** [comment]
:+1:

**Pull request merged**